cmake/MiscCheck.cmake

```diff
@@ -142,30 +142,14 @@ cmake_pop_check_state()
 
 # ---[ Check if the compiler has AVX/AVX2 support. We only check AVX2.
 if(NOT INTERN_BUILD_MOBILE)
-  cmake_push_check_state(RESET)
-  if(MSVC AND NOT CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    set(CMAKE_REQUIRED_FLAGS "/arch:AVX2")
-  else()
-    set(CMAKE_REQUIRED_FLAGS "-mavx2")
-  endif()
-  CHECK_CXX_SOURCE_COMPILES(
-      "#include <immintrin.h>
-      int main() {
-        __m256i a, b;
-        a = _mm256_set1_epi8 (1);
-        b = a;
-        _mm256_add_epi8 (a,a);
-        __m256i x;
-        _mm256_extract_epi64(x, 0); // we rely on this in our AVX2 code
-        return 0;
-      }" CAFFE2_COMPILER_SUPPORTS_AVX2_EXTENSIONS)
-  if(CAFFE2_COMPILER_SUPPORTS_AVX2_EXTENSIONS)
```

Comment:
This variable that you've removed is actually used in caffe2, which is causing the build to fail.

Reply:
Makes sense. I'm pushing a fix.